Apps/WebClient/src/ClientApp/src/components/calendar/calendar.vue

```diff
@@ -131,9 +121,7 @@ export default class CalendarComponent extends Vue {
                     :available-months="availableMonths"
                 />
             </b-col>
-            <b-col v-if="isNoteEnabled" col cols="auto">
-                <add-note-button text-breakpoint="md" class="p-2" />
-            </b-col>
+            <slot name="add-note" />
```

Comment:
nit: The space is not _necessarily_ for notes since a different component could fit in the right side

Reply:
A concern for future Mike!